api/api/settings.py

```diff
@@ -214,6 +214,17 @@
 WATCHDOG_SLAVES = os.environ.get('DESECSTACK_WATCHDOG_SLAVES', '').split()
 WATCHDOG_WINDOW_SEC = 600
 
+# Prometheus
+# As per https://github.com/korfuri/django-prometheus/blob/master/documentation/exports.md while we can't use
+# PROMETHEUS_METRICS_EXPORT_PORT_RANGE due to https://github.com/korfuri/django-prometheus/issues/215
+try:
+    import prometheus_client
+    import uwsgi
+    prometheus_client.values.ValueClass = prometheus_client.values.MultiProcessValue(
+        _pidFunc=uwsgi.worker_id)
+except ImportError:
+    pass  # not running in uwsgi
```

Comment:
should this be narrower to avoid false silence of errors?

Reply:
tried to do something smarter in the fixup commit - ok now?

Reply:
right, the clause was too large. lgtm!